assets/messages.yml

```
@@ -316,7 +316,9 @@ messages:
       name: Duplicate (private)
       message: |-
         *Thank you for your report!*
-        We're actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
+        We''re actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
```

```suggestion
        We're actually already tracking this issue as *%s%* (private), so this ticket is being resolved and linked as a *duplicate*.
```

Same as https://github.com/mojira/helper-messages/pull/42#discussion_r426697671